### DISCUSSION TITLE
Why am I seeing 404 error on all meeting pages?

### DISCUSSION THREAD

**Discussion by Area82** [code4recovery/12-step-meeting-list#527] [Q&A] [answered]
### Contact Details

webmaster@area82aa.org

### Website With Issue

https://area82aa.org/meetings/

### What happened?

Suddenly all of my meeting pages no longer exist - individual meeting pages and the main meetings page. I've tried deactivating all of my plug ins as well as my themes and none of them are causing this bug. 

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari

### Relevant log output/errors

```shell
Failed to load resource: the server responded with a status of 404 ()
```

**Comment by tech2serve** [comment, marked as answer]
This is most likely not a bug with TSML. The two most common reasons for this issue are permalinks or a change by the webmaster or host provider that affected the links.

For the permalinks, it is often not clear why Wordpress experiences this, but luckily it is an easy fix. Navigate to Settings->Permalinks, and then simply scroll down and click "Save Changes". This refreshes Wordpress links that affect TSML, and often resolves page not found issues.

If that doesn't fix your issue, take a look at why your site is loading from a "temp.domains"

<img width="538" alt="image" src="https://user-images.githubusercontent.com/31904316/133624556-018dcece-879d-438c-84e9-cbae9b8f1a87.png">

I'm not saying this won't work, or that it would cause problems, but it strikes me as unorthodox and does have your site going from secure pages to unsecured. There may be a better way to arrange you data.

Let me know if these steps resolve your issues.

**Reply by joshreisner** [reply]
+1 to what @tech2serve said above about permalinks. Looks like your site is expecting "post name" to be selected under common permalinks settings.

Regarding the domain, one really easy way to enforce the correct URL on your site is to add these lines to your `wp-config.php`:
```
define( 'WP_HOME', 'https://area82aa.org' );
define( 'WP_SITEURL', 'https://area82aa.org' );
```

**Reply by Area82** [reply]
Thank you both, the "temp.domains" was just a link i forgot to update. Thank you for pointing that out. The permalink solution you mentioned worked to fix the 404 errors. I appreciate the help!
